src/ErrorHandler.php

```diff
@@ -19,7 +20,7 @@ final class ErrorHandler
     /**
      * The default amount of bytes of memory to reserve for the fatal error handler.
      */
-    private const DEFAULT_RESERVED_MEMORY_SIZE = 10240;
+    public const DEFAULT_RESERVED_MEMORY_SIZE = 10240;
```

Comment:
Mainly for the first reason you said. I don't know if it will be ever useful though, probably not since if you call the `registerOnceFatalErrorHandler()` method and you want to specify its only argument then probably you won't use the default value...